Form-Controls/index.html

```diff
@@ -1,27 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>T-Shirt Order Form</title>
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>Order Your T-Shirt</h1>
+      <p>Fill out the form below to select your T-shirt preferences.</p>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <!-- Customer Name -->
+        <div>
+          <label for="customer-name">Name:</label>
+          <input
+            type="text"
+            id="customer-name"
+            name="customer-name"
+            minlength="2"
```

Comment:
This is good (`minlength="2"`) - there's a good chance here that usually this will catch people who by accident have not entered their name.